docs/issue-to-earn.mdx

```diff
@@ -0,0 +1,319 @@
+        },
+        // highlight-end
+        "expirationDate": 1893484800
+      }
+    ]
+  }
+  ```
+
+  Format the credential data according to the `properties` attribute of the credential schema. In this case, the schema is for a `SsnCredential` which can found found at [`https://schema.verified.inc/jsonSchema/SsnCredential`](https://schema.verified.inc/jsonSchema/SsnCredential). 
+
+  More information about credential schemas can be found in the [Schema](/schema) section.
+
+  ✅ **_ You're done!_**
+
+  > For more detailed information on how and why to implement Free IDV, see our [Free IDV Guide](/issuance-guide).
+
+</TabItem>
+  
+<TabItem value="1-click" label="1-Click Only">
+
+  **You need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, 1-Click IDV is available.
+
+  1. **Call [`/hasMatchingCredentials`](/api-overview/#check-user-credentials)** with the user's email and/or phone and the credential type(s) you need.
+
+    ```typescript title="Example Response Body"
+    {
+        "match": true,
+        "url": "https://wallet.verified.inc/request/9f2053fc-cc4c-41cb-8d14-7d3ccf6167eb"
+    }
+  ```
+
+  <details>
+    <summary>What counts as a match?</summary>
+    <p>
+      When you call <code>/hasMatchingCredentials</code>, you include one or more credential requests. For each credential request, you specify:
+      <ul>
+        <li>
+          the type
+        </li>
+        <li>
+          (optional) which issuers are acceptable
+        </li>
+        <li>
+          (optional) whether the credential is required
+        </li>
+      </ul>
+    </p>
+    <p>
+      There's a match if, for each credential request for a required credential, the user has a credential of the correct type from an acceptable issuer.
+    </p>
+  </details>
+    
+  The user will either have matching credentials, or they won't. If they do, continue to step (2). If they don't, you're done with Verified Inc. and should take the user through your existing verification flow. <i>If you would like your existing verification flow verification costs to be covered, please look into the <b>Both</b> section of this quick start, which includes both Free IDV and 1-Click IDV.</i>
+
+  2. **Direct the user to the `url`** the response body contains with an additional `redirectUrl` query param that you define which will serve as the ultimate landing page after the user completes 1-Click IDV from the Verified Inc. web wallet. For example the final url you need to redirect the user to is:
+
+  ```text title="Example Redirect to Verified Inc. Wallet URL"
+  https://wallet.verified.inc/request/9f2053fc-cc4c-41cb-8d14-7d3ccf6167eb?redirectUrl=https://acme-bank.co/sign-up/complete
+  ```
+
+  3. **Capture the `sharedCredentialsUuid` query param** from the `redirectUrl` page you defined in step (2). To continue the example in step (2), the final url the user will be redirected to will be something like this:
```

Comment:
We probably should just reuse an explanation component here explaining the various ways to get user credentials (and use across both flavors of Guides).